built_value_generator/lib/built_value_generator.dart

```diff
@@ -27,9 +27,12 @@ class BuiltValueGenerator extends Generator {
           serializerSourceLibrary.hasSerializers) {
         result.writeln(serializerSourceLibrary.generateCode());
       }
-    } catch (e, st) {
+    } on InvalidGenerationSourceError catch (e, st) {
```

Comment:
The difference is whether we call "e.message"; only InvalidGenerationSourceError has a "message".